src/lib.rs

```diff
@@ -14,11 +14,12 @@
 //! This crate implements the ACE-OAuth
 //! (Authentication and Authorization for Constrained Environments using the OAuth 2.0 Framework)
 //! framework as defined in [RFC 9200](https://www.rfc-editor.org/rfc/rfc9200).
-//! Key features include CBOR-(de-)serializable data models such as [`AccessTokenRequest`](crate::endpoints::token_req::AccessTokenRequest),
+//! Key features include CBOR-(de-)serializable data models such as [`AccessTokenRequest`],
```

Comment:
Possibly. I removed those link targets because the linter (unsure if it was clippy or the one provided by CLion) recommended removing them as they were redundant.